### DISCUSSION TITLE
how did you get Download Badge to work for IronOS?

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1602] [Q&A] [answered]
Any ideas?

I went to here https://shields.io/category/downloads

although I see the preview works if I type in "ralim" "ironos"

I can not make it work for "builder555" "PineSAM"  and I'm trying to set up download badge for it.

![image](https://user-images.githubusercontent.com/97197236/222297534-36067faf-8d74-42de-b8d0-ee2dc748d993.png)

![image](https://user-images.githubusercontent.com/97197236/222297646-c9eb8878-8be8-4c8b-b5c9-8437996b0ab6.png)

**Comment by Ralim** [comment]
I dont know for sure; but it could be either not enough downloads _or_ metrics being behind.

**Comment by River-Mochi** [comment]
it's been a couple days now and it doesn't work. still says download is "0"
if anyone has any ideas for why the Download Badge doesn't  work  at  [github page](https://github.com/builder555/PineSAM)  please let me know
https://github.com/builder555/PineSAM

**Reply by Ralim** [reply]
Most likely not enough downloads or github just isn't tracking them tbqh.

Ironos metrics only started a while after I made it.

**Comment by River-Mochi** [comment, marked as answer]
I  found out that the API for PineSAM does not have any **download_count** in it even though there are zip releases. 
 PineSAM is all python script and does not contain anything in Releases except the two default "source code" zip packages.

Basically it needs a named zip in "Releases" besides the default ones called Souce code.zip and Source code.tar.gz.  Only then does github count it with download_count similar to how it's shown in IronOS releases API.
https://api.github.com/repos/ralim/ironos/releases
